docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+</Tabs>
+
+<!---  Tabs end here -->
+
+
+### Structured Directory Validation
+One method to quickly complete validation is to structure your data in a pre-determined format that `seq-tools` is expecting. Under a submission directory, compile each submission as a folder, with the JSON payload and the associated data files in the same folder. 
+
+Each submission folder **must** contain:
+- The metadata payload as a `.json` file. Only 1 `.json` file must be present in the submission folder.
+- All molecular data files associated to that payload, or symlinks to their location.
+
+For example: 
+![Example Directory Structure](/assets/submission/molecular_submission_validation_directory.png)
+
+Once your data is ready, use `seq-tools` from the submission directory to validate all of the data: 
+
+<!---  Tabs start here -->
+
+<Tabs
+  groupId="operating-systems"
+  defaultValue="client"
+  values={[
+    { label: 'Client', value: 'client', },
+    { label: 'Docker', value: 'docker', },
+  ]
+}>
+<TabItem value="client">
+
+```bash
+seq-tools validate submission-directory/*/*.json  
+```
+
+</TabItem>
+<TabItem value="docker">
+
+```bash
+cd ..
+seq-tools-in-docker validate -d submission-directory/*/*.json 
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+
+You will see an interactive prompt of the results of validation as the tool runs.  Once completed, view the details of the validation report:
+
+```bash
+cat validation_report.PASS.jsonl | jq . | less
+cat validation_report.INVALID.jsonl | jq . | less
+```
+### Scriptable Location Validation
+
+If you do not want to use the structured directory, you can use the '-d' option if data files are located in a different directory than where the metadata payloads are listed.
+
+```bash
+seq-tools validate -d ../sequencing-data-directory/ metadata_file_only/sequencing_experiment_payload.json
+```
+Or validate all metadata payloads in a directory at once using a wildcard: 
+```
+seq-tools validate -d ../sequencing-data-directory/ submission-payload-directory/*/*.json  
+seq-tools-in-docker validate -d ../sequencing-data-directory/ submission-payload-directory/*/*.json
+```
+
+You can use this method to build your own script to validate payloads in different locations. 
+
+### Understanding the Validation Report
+Validation report summaries can be found in the top level submission directory where you ran `seq-tools`.  The report.  Reports are separated into three categories:
+- **PASS:** This status indicates that these payload(s) are ready for submission. 
+- **PASS-with-WARNING:** This status indicates that these payload(s) are ready for submission, however there may be a parameter you want to double check before submission. 
+- **INVALID:** This status indicates that these payload(s) are _not_ ready for submission. 
+
+In the **INVALID** summary report, each reason for failure is listed. Errors must be fixed before you attempt to submit this payload.  To view all reports that are generated:
```

Comment:
suggestion to complete this sentence....
"To view all reports that are generated, use the following commands:"

Reply:
there are technically other ways to do this - we are just being helpful.  i dont want it to seem like this is the only way, so i will leave it how it is